services/src/handlers/wcs.rs

```diff
@@ -466,7 +474,12 @@ mod tests {
                 <ows:Title>Web Coverage Service</ows:Title>
                 <ows:ServiceType>WCS</ows:ServiceType>
                 <ows:ServiceTypeVersion>1.1.1</ows:ServiceTypeVersion>
+                <ows:Fees>NONE</ows:Fees>
+                <ows:AccessConstraints>NONE</ows:AccessConstraints>
             </ows:ServiceIdentification>
+            <ows:ServiceProvider>
+                <ows:ProviderName>Provider Name</ows:ProviderName>
```

Comment:
Geo Engine?

Reply:
This is the operator of the WCS endpoint. We wouln't want to have Geo Engine there if somebody else hosts it.